files/templates/mbta/login/register.ftl

```diff
@@ -91,6 +95,36 @@
 	                           value="${(register.formData.email!'')}" autocomplete="email"
 	                           aria-invalid="<#if messagesPerField.existsError('email')>true</#if>"
 	                    />
+	                    
+	                    <#if messagesPerField.existsError('email')>
+	                        <div class="instructions-container">
+		                        <span id="input-error-lastname" class="${properties.kcInputErrorMessageClass!}" aria-live="polite">
+		                            ${kcSanitize(messagesPerField.get('email'))?no_esc}
+		                        </span>
+		                    </div>
+	                    </#if>
+	            </div>
+	            
+	            <div class="form-group">
+	                    <label for="user.attributes.phone" class="form-input-label<#if messagesPerField.existsError('user.attributes.phone_number')> label-error</#if>">${msg("user.attributes.phone")} <span>${msg("user.attributes.phone.span")?no_esc}</span></label>
+	                    <div class="input-group">
+	                    	<label class="visually-hidden" for="user.attributes.areacode">${msg("countryCode")}</label>
+		                    <select id="user.attributes.areacode" class="form-select<#if messagesPerField.existsError('user.attributes.phone_number')> input-error</#if>" name="user.attributes.areacode">
```

Comment:
question: can you say more about this attribute? I don't think I saw it when I was looking at the attributes a couple weeks ago.